.jenkins/deploy/jjb.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+#
+# Create a new Job in OCP that runs the jbb-validate container once. This
+# script will wait for completion of the validation, and uses the result of the
+# container to report the status.
+#
+# Usage:
+# - arguments to the script can be "validate" or "deploy"
+# - the GIT_REF environment variable is injected in the batch job so that it
+#   can use a particular GitHub PR
+#
+
+# error out in case a command fails
+set -e
+
+CMD="${1}"
+GIT_REF=${GIT_REF:-"master"}
+
+get_pod_status() {
+	oc get "pod/${1}" --no-headers -o=jsonpath='{.status.phase}'
+}
+
+case "${CMD}" in
+	"validate")
+		;;
+	"deploy")
+		;;
```

Comment:
What are those empty cases? Is it intentional? 
Are they only to validate that the arg is correct?

Reply:
yes, only to validate the argument